From 15840a46703410a5101835df67ff077904ce1bcd Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Wed, 7 Feb 2018 16:26:53 +0800
Subject: [PATCH] Revert "soc: rockchip: rockchip_pm_config: set the real pin
 number to ATF"

This reverts commit ef06da289014f33b795f639f1e1db84d4cfefb96.

Change-Id: Ide586a9f1aa079b9d29568ed0c88235f58ccfe9a
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 drivers/soc/rockchip/rockchip_pm_config.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/soc/rockchip/rockchip_pm_config.c b/drivers/soc/rockchip/rockchip_pm_config.c
index ece90ca19cc5..8ecf3f884c98 100644
--- a/drivers/soc/rockchip/rockchip_pm_config.c
+++ b/drivers/soc/rockchip/rockchip_pm_config.c
@@ -59,7 +59,6 @@ static int __init pm_config_init(struct platform_device *pdev)
 	enum of_gpio_flags flags;
 	int i = 0;
 	int length;
-	int pin_num;
 
 	match_id = of_match_node(pm_match_table, pdev->dev.of_node);
 	if (!match_id)
@@ -105,11 +104,9 @@ static int __init pm_config_init(struct platform_device *pdev)
 							     &flags);
 			if (!gpio_is_valid(gpio_temp[i]))
 				break;
-			/* get the real pin number */
-			pin_num = gpio_temp[i] - ARCH_GPIO_BASE;
 			sip_smc_set_suspend_mode(GPIO_POWER_CONFIG,
 						 i,
-						 pin_num);
+						 gpio_temp[i]);
 		}
 	}
 	sip_smc_set_suspend_mode(GPIO_POWER_CONFIG, i, PM_INVALID_GPIO);
-- 
2.35.3

